packages/backend-core/src/db/couch/DatabaseImpl.ts

```diff
@@ -371,11 +371,15 @@ export class DatabaseImpl implements Database {
     return this.performCall(() => {
       return async () => {
         const response = await directCouchUrlCall(args)
-        const json = await response.json()
-        if (response.status > 300) {
-          throw json
+        if (response.status >= 300) {
+          const text = await response.text()
```

Comment:
Will this always be text?

Reply:
Correct me if I'm wrong but I thought `.text()` just got the plain old body contents without trying to parse it, so I thought it was always safe to use.

Reply:
Yes, you are right!